blog/2020-05-08-AIVersusCovid-19.adoc

```diff
@@ -0,0 +1,135 @@
+= AI versus Covid-19: How Java helps nurses and doctors in this fight
+:page-interpolate: true
+:awestruct-author: Christopher-Chianelli
+:awestruct-layout: blogPostBase
+:awestruct-tags: [use case]
+
+Scheduling a hospital is tough. To maintain employees' happiness and health,
+you need to take into account sick days, days off requests, and assure
+the employees have ample time between shifts. This is doubly so in a pandemic,
+where you need to minimize exposure to the new disease and have nurses with
+different risks. Let's take a look at how we can quickly and efficiently
```

Comment:
> ... you need to ... have nurses with different risks.

I don't understand this. Risk of what? Did you mean nurses with different skills? That would make sense to me.

Reply:
Ex: older nurses, nurses with diabetes, etc. Or worded differently: Different susceptibility to COVID-19.